README.md

```diff
@@ -473,6 +474,22 @@ for the possible value of `level`.
 
 [Back to TOC](#table-of-contents)
 
+resty.kong.peer_conn.get\_last\_peer\_connection\_cached
+----------------------------------
+**syntax:** *resty.kong.peer_conn.get_last_peer_connection_cached()*
+
+**context:** *balancer_by_lua*
+
+**subsystems:** *http*
+
+Retrieves whether the connection used in the previous attempt came from the upstream connection pool when the next_upstream retrying mechanism is in action.
+
+Possible result are as follows:
+
+- `0`: The connection was not reused from the upstream connection pool, it means that a new connection was created with the upstream in the previous attempt.
```

Comment:
Let's not use `0` and `1`, this kind of method should return `true` or `false` instead.

Also, could you explain why is this method useful? Like:

> It is useful for determineing xxxx

Reply:
add more desc